spice/cfg/tests-variants.cfg

```diff
@@ -165,6 +165,22 @@ variants:
                             ticket_send = 12456
                             include join.cfg
 
+                - 527185:
+                    # only for qemu-kvm-rhev testing
+                    spice_proxy = http://[2620:52:0:2249:21a:4aff:fe01:46a3]:80
+                    include join.cfg
+
+                - 525961:
+                    spice_proxy = http://10.34.73.1
```

Comment:
Cartesian config allows to use variables.

We must define this value once at tests.cfg and use  instead specifying IP each time.

Lines to replace:

g 10.34.73.1
./spice-base.cfg:987:###        spice_proxy = http://10.34.73.1:3128
./spice-base.cfg:995:###        spice_proxy = http://10.34.73.1
./tests-variants.cfg:129:                    spice_proxy = http://10.34.73.1:3128
./tests-variants.cfg:151:                    spice_proxy = http://10.34.73.1
./tests-variants.cfg:702:                    spice_proxy = http://10.34.73.1:3128

Define variables

http_proxy_ip = 10.34.73.1
http_proxy_port = 3128

And use them